mkosi/__init__.py

```diff
@@ -1478,6 +1478,10 @@ def invoke_repart(state: MkosiState, skip: Sequence[str] = [], split: bool = Fal
     for fs, options in state.installer.filesystem_options(state).items():
         env[f"SYSTEMD_REPART_MKFS_OPTIONS_{fs.upper()}"] = " ".join(options)
 
+    for option, value in state.environment.items():
+        if option[0:28] == "SYSTEMD_REPART_MKFS_OPTIONS_":
```

Comment:
Use startswith() here.

Reply:
Now I finally see why I broke the type checker when using `startswith()`.